src/components/modal/modal-component.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { ReactComponent as InfoIcon } from 'icons/info.svg';
+
+import { Modal, Loading } from 'he-components';
+import styles from './modal-styles.module.scss';
+
+const ModalMetadata = ({ isOpen, handleClose, loading, title, metadata = {} }) => {
+  return (
+    <Modal
+      onRequestClose={handleClose}
+      isOpen={isOpen}
+      theme={styles}
+    >
+      <h2>{title}</h2>
+      {
+        loading ? <Loading height={200} /> : (
+          <>
+            {
```

Comment:
Can we move `metadata` condition to the top somehow, without repeating it below?

Reply:
i will be changing this file anyway once we have metadata, i'll leave it like this for now